### DISCUSSION TITLE
CATs draft standard has gone live!

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#9295] [Announcements]
Our CAT1 standard  (Chia Asset Tokens) has gone live in draft form! While this remains subject to change until fully released, we do not anticipate significant or breaking changes, but cannot 100% rule them out. That said, they are now live on-chain and available to experiment with via our new Lightweight Wallet Beta!

To learn all about the new CAT standard, check out this blog post here: https://www.chia.net/2021/11/15/the-CATs-out-of-the-bag.en.html

And you can obtain the beta version of the new lightweight wallet on our downloads page at: https://www.chia.net/download/

_WARNING: Do __NOT__ install this wallet on an active farming machine. If you do, then it will overwrite your current install settings and YOU WILL NO LONGER BE FARMING! In case that wasn’t clear enough,_

___DO NOT INSTALL THIS ON A MACHINE THAT IS ACTIVELY FARMING!___

_That said, our light wallet uses an electrum-style protocol to reduce syncing from hours to just minutes, or even seconds. It’s also our first wallet implementation supporting Chia Asset Tokens (CATs). We'll make this available to be used as part of the standard Chia package on the same machine as a farmer in a future release._

_If you do happen to ignore the above warning and overwrite your farmer, don't panic as you can still get it back. Data for your farm and full node are not deleted and live in a separate folder from the light wallet. You can always re-run the Chia farming installer to return your configurations to farming, which will delete your light wallet._

**Comment by Bafff** [comment]
Hi!

Can this wallet be installed on the harvester?

**Comment by SlugFiller** [comment]
Question. If CATs do their own bookkeeping, why do they need a melt value? Wouldn't it be better to store the value as a curry, thereby making it possible to mint an infinite amount of CATs without spending a single mojo?

**Comment by thechiaplot** [comment]
3113CAFA732BDBB9E7C5A0998CBA4F189FB2E5EC4A52D1B02494DF6882D42758
